packages/lodestar-validator/src/util/beaconChain.ts

```diff
@@ -0,0 +1,84 @@
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
```

Comment:
makes me sad :sob:

Reply:
I know :(

Reply:
:(

Reply:
wait whats going on here...?

Reply:
haha. I had to copy few state transition methods in this file as we cannot easily rip state transition into separate module